filter.js

```diff
@@ -162,7 +164,77 @@ let filter = {
   
       // Log the words array
       //console.log(filter.words);
+            
     }
   });
-  
+
+//Searching for challenges titles that matches the keyword
+document.getElementById("searchBox").addEventListener("keyup", searchRoomByKeyW, searchRoomByDesc);
```

Comment:
I don't believe this is a valid use of `addEventListener()`. The documentation for `addEventListener()` is here: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

You are passing three arguments to the method, and when three arguments are passed, the third one can either be a boolean to indicate to use the "capture phase" instead of the "bubble phase", or an object with various settings for the event listener.

You are instead passing two functions. This means that the second function will never be called. In this case, it means that your `searchRoombyDesc` function is never used and hence I believe that searching for description does not work.